seinfoflow

```diff
@@ -75,11 +78,28 @@ elif args.verbose:
 else:
     logging.basicConfig(level=logging.WARNING, format='%(message)s')
 
+conditionals = None
+if args.conditionals == 'default':
+    conditionals = {}
+elif args.conditionals is not None:
+    conditionals = {}
+    for conditional in args.conditionals.split():
+        try:
+            key, value = conditional.split(':')
+            if value.lower() == 'true':
+                conditionals[key] = True
+            elif value.lower() == 'false':
+                conditionals[key] = False
+            else:
+                parser.error("Conditional value must be true or false.")
```

Comment:
I'd be ok with being a little looser on the checking, i.e. `if value.lower() == "true"` then the Boolean is considered `True`, and then in all other cases, it is `False`. If you make that change (I'm not requiring this change), please make sure to explain this in the option's help text.